components/predict-form/Predict.tsx

```diff
@@ -371,56 +375,96 @@ export function Predict({
-                      ? "What day will my project be done by?"
-                      : "Will I finish my project by Friday?"
-                  }
-                  maxRows={15}
-                  onChange={(e) => {
-                    updateTagsPreview(e.currentTarget.value)
-                    void onChangeQuestion(e)
-                  }}
-                  onKeyDown={onEnterSubmit}
-                  defaultValue={questionDefaults?.title}
-                  onMouseDown={(e) => e.stopPropagation()}
-                  {...registerQuestion}
-                  onFocus={() => setQuestionInFocus(true)}
-                  onBlur={() => setQuestionInFocus(false)}
-                  ref={mergeRefs([textAreaRef, formRef])}
-                />
-
-                {showQuestionSuggestionsButton && (
-                  <button
-                    tabIndex={-1}
+                >
+                  <TextareaAutosize
                     className={clsx(
-                      "inline-flex align-middle justify-center text-center btn btn-circle aspect-square absolute right-3 top-[calc(50%+1rem)] hover:opacity-100 !-translate-y-1/2",
-                      showSuggestions ? "btn-active" : "btn-ghost",
-                      !!question && !showSuggestions
-                        ? "opacity-20"
-                        : "opacity-80",
-                      small && "btn-xs px-5",
+                      "w-full resize-none rounded-md p-4 -mb-2 border-b bg-white",
+                      "focus:outline-indigo-700 placeholder:text-neutral-400",
+                      small ? "text-md" : "text-xl",
                     )}
-                    onClick={(e) => {
-                      setShowSuggestions(!showSuggestions)
-                      e.preventDefault()
+                    placeholder={
+                      placeholder ||
+                      questionType === QuestionType.MULTIPLE_CHOICE
+                        ? "What day will my project be done by?"
+                        : "Will I finish my project by Friday? "
+                    }
+                    maxRows={15}
+                    onChange={(e) => {
+                      updateTagsPreview(e.currentTarget.value)
+                      void onChangeQuestion(e)
                     }}
-                  >
-                    <LightBulbIcon
-                      height={16}
-                      width={16}
-                      className="shrink-0"
+                    onKeyDown={onEnterSubmit}
+                    defaultValue={questionDefaults?.title}
+                    onMouseDown={(e) => e.stopPropagation()}
+                    {...registerQuestion}
+                    onFocus={() => setQuestionInFocus(true)}
+                    onBlur={() => setQuestionInFocus(false)}
+                    ref={mergeRefs([textAreaRef, formRef])}
+                  />
+                  <div className="flex flex-row items-center p-2 sm:px-4 gap-1 z-20">
+                    <TagsSelect
```

Comment:
minor: It's a bit strange that the add new icon is not full width, not a big deal though. Maybe could make the background of the empty space be the same grey so it looks a bit more like it spans the full width?
<img width="707" alt="image" src="https://github.com/user-attachments/assets/4c4efe3b-98ec-40ee-800d-714d5e79ea89">